templates/parachain/zombienet/tests/smoke.rs

```diff
@@ -0,0 +1,49 @@
+//! This test is setup to run with the `native` provider and needs these binaries in your PATH
+//! `polkadot`, `polkadot-prepare-worker`, `polkadot-execute-worker`, `parachain-template-node`.
+//! You can follow these steps to compile and export the binaries:
+//! `cargo build --release -features fast-runtime --bin polkadot --bin polkadot-execute-worker --bin
+//! polkadot-prepare-worker`
```

Comment:
```suggestion
//! `cargo install --git https://github.com/paritytech/polkadot-sdk/ --tag polkadot-stable2407
//! polkadot polkadot-execute-worker polkadot-prepare-worker`
```

Ideally we could automatically replace the tag here as the template updates